providers/curation/github.js

```diff
@@ -144,6 +144,15 @@ class GitHubCurationService {
       state = 'error'
       description = `Invalid curations: ${invalidCurations.map(x => x.path).join(', ')}`
       this.logger.error(description, invalidCurations)
+
+      let error_string = 'We discovered some errors in this curation when validating it:\n\n'
+
+      for (const invalid_curation of invalidCurations) {
```

Comment:
TIL that `for ... of` exists! I've been using `forEach` and `for ... in`, but never heard of using `of` before.

Another pattern I've seen in CD is to use `map` to build the string, but I don't think it makes a difference here.